src/website/utils/validators.py

```
@@ -0,0 +1,61 @@
+    if '.' not in file.name:
+        return
+
+    allowed_extensions = ['jpg', 'jpeg', 'png', 'gif', 'bmp']
+    extension_validator = FileExtensionValidator(allowed_extensions)
+
+    try:
+        extension_validator(file)
+    except ValidationError:
+        raise ValidationError(
+            f"Unsupported file extension. Allowed extensions are: {', '.join(allowed_extensions)}."
+        )
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)